node.go

```diff
@@ -166,12 +164,20 @@ func (node Node) restoreMetadata(path string) error {
 }
 
 func (node Node) RestoreTimestamps(path string) error {
-	var utimes = []syscall.Timespec{
+	var utimes = [...]syscall.Timespec{
 		syscall.NsecToTimespec(node.AccessTime.UnixNano()),
 		syscall.NsecToTimespec(node.ModTime.UnixNano()),
 	}
 
-	if err := syscall.UtimesNano(path, utimes); err != nil {
+	if node.Type == "symlink" {
+		if err := node.restoreSymlinkTimestamps(path, utimes); err != nil {
+			return errors.Annotate(err, "UtimesNano")
```

Comment:
I feel like this annotation should happen in `restoreSymlinkTimestamps`, not here